scripts/prep-spd-roster.py

```diff
@@ -0,0 +1,52 @@
+import pandas as pd
+import sys
+
+from pandas.core.frame import DataFrame
+
+
+def main():
+    date = sys.argv[1]
+    in_csv = sys.argv[2]
+    out_csv = sys.argv[3]
+
+    # Import CSV passed to script
+    df = pd.read_csv(in_csv)
+
+    # Rename columns to better formatting
+    column_names = {
+        'Name': 'full_name',
+        'Badge_Num': 'badge',
+        'Title_Description': 'title',
+        'Unit': 'unit',
+        'Unit_Description': 'unit_Description'

```

Comment:
Should the second "description" also be lowercase?

Reply:
It sure should be thank you